lib/jbuild

```diff
@@ -0,0 +1,37 @@
+(* -*- tuareg -*- *)
+#require "unix"
+
+let flags = function
+  | [] -> ""
+  | pkgs ->
+    let cmd = "ocamlfind ocamlc -verbose" ^ (
+        List.fold_left (fun acc pkg -> acc ^ " -package " ^ pkg) "" pkgs
+      ) in
+    let ic = Unix.open_process_in
+        (cmd ^ " | grep -oEe '-ppx (\"([^\"\\]|\\.)+\"|\\w+)'")
+    in
+    let rec go ic acc =
+      try go ic (acc ^ " " ^ input_line ic) with End_of_file -> close_in ic; acc
+    in
+    go ic ""
+
+
+let rewriters = ["ppx_deriving_rpc"]
+let flags = flags rewriters
+
+
+let () = Printf.ksprintf Jbuild_plugin.V1.send {|
+(jbuild_version 1)
+
+(library (
+  (name tapctl)
+  (public_name xapi-tapctl)
+  (wrapped false)
+  (flags (:standard -w -37-39 %s))
+  (libraries (
+    rpclib
+    threads
+    forkexec
```

Comment:
Looking at `lib/tapctl.ml` it seems that we should also add `xapi_stdext_unix`, `xapi_stdext_std`, `xapi_stdext_threads` and `xapi_stdext_monadic` here